deps/Makefile

```diff
@@ -36,7 +36,7 @@ libssl: libssl/openssl/libssl.a
 
 libhttpserver/libhttpserver/build/src/.libs/libhttpserver.a: libmicrohttpd/libmicrohttpd/src/microhttpd/.libs/libmicrohttpd.a
 	cd libhttpserver && rm -rf libhttpserver-master_20191121 || true
-	cd libhttpserver && tar -zxf libhttpserver-master_20191121.tar.gz
+	cd libhttpserver && tar -zxf libhttpserver-master_20191121.tar.gz && mv libhttpserver-master_20191121 libhttpserver
```

Comment:
This doesn't seem correct, as `libhttpserver` exists as a [symbolic link](https://github.com/sysown/proxysql/blob/v2.0.9/deps/libhttpserver/libhttpserver#L1)

Reply:
You're right, it was my fault messing up with the link and breaking it